roles/master-prerequisites/tasks/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- block:
+  - name: Install git
+    package:
+      name: "{{ item }}"
+      state: latest
+    with_items:
+    - git
```

Comment:
Since this is only a single package, it might make sense to drop the loop.

The block seems unnecessary as well, since there is only the single task.